### DISCUSSION TITLE
Cannot find package while trying  to install tool

### DISCUSSION THREAD

**Discussion by amogh0503** [projectdiscovery/nuclei#777] [Q&A] [answered]
When installing go get -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei
Getting error

github.com/pierrec/lz4 (download)
package github.com/pierrec/lz4/v4: cannot find package "github.com/pierrec/lz4/v4" in any of:
        c:\go\src\github.com\pierrec\lz4\v4 (from $GOROOT)
        C:\Users\amogh\go\src\github.com\pierrec\lz4\v4 (from $GOPATH)
github.com/google/go-github (download)
package github.com/google/go-github/v32/github: cannot find package "github.com/google/go-github/v32/github" in any of:
        c:\go\src\github.com\google\go-github\v32\github (from $GOROOT)
        C:\Users\amogh\go\src\github.com\google\go-github\v32\github (from $GOPATH)

**Comment by ehsandeep** [comment, marked as answer]
@amogh0503 it's an issue with your go setup, you can try with additional installation method to install nuclei - https://nuclei.projectdiscovery.io/nuclei/get-started.html#nuclei-installation
